packages/react/src/components/Toolbar/ToolbarMenu.tsx

```diff
@@ -70,16 +71,38 @@ class ToolbarMenu extends UIComponent<ToolbarMenuProps> {
     variables: mergeComponentVariables(variables, predefinedProps.variables),
   })
 
+  handleRadioGroupOverrides = variables => (predefinedProps: ToolbarMenuRadioGroupProps) => ({
+    onItemClick: (e, itemProps) => {
+      _.invoke(predefinedProps, 'onItemClick', e, itemProps)
+      _.invoke(this.props, 'onItemClick', e, itemProps)
+    },
+    variables: mergeComponentVariables(variables, predefinedProps.variables),
+  })
+
   renderItems(items, variables) {
     const itemOverridesFn = this.handleItemOverrides(variables)
     const dividerOverridesFn = this.handleDividerOverrides(variables)
+    const radioGroupOverrides = this.handleRadioGroupOverrides(variables)
 
     return _.map(items, item => {
       const kind = _.get(item, 'kind', 'item')
-      if (kind === 'divider') {
-        return ToolbarMenuDivider.create(item, { overrideProps: dividerOverridesFn })
+
+      switch (kind) {
+        case 'checkbox':
```

Comment:
just noticed that we call it `toggle` in the `Toolbar` component. We should probably have the same naming across these two